biomage/rds/run.py

```diff
@@ -99,12 +99,12 @@ def run_rds_command(
 
     password = None
 
-    local_port = local_port or 5432
-
     if input_env == "development":
         password = "password"
         local_port = local_port or 5431
```

Comment:
It's to fix this, it was setting port 5432 for development because the line at 102 made sure that local_port was always already filled